src/main/resources/application.yaml

```diff
@@ -164,4 +164,9 @@ feature-flags:
   # noinspection SpringBootApplicationYaml
   hearings-api: ${HEARINGS_API_FLAG:true} # will be used in future releases, remove noinspection once in use
 
+case-links:
+  linked: ${CASE_LINKED_TEMPLATE_ID:7e8c6fc1-520a-41cd-a85c-9d3e0b22b282}
+  unlinked: ${CASE_UNLINKED_TEMPLATE_ID:8099d7b8-1703-40c3-95e6-464a3ac776cd}
```

Comment:
We have a template section further up in this file, is it worth putting these under that section? ie `template.case-links.linked`

Reply:
Done